packages/fxa-auth-server/lib/senders/emails/layouts/subscription/index.mjml

```diff
@@ -77,6 +77,11 @@
               &nbsp;&nbsp;&bull;&nbsp;&nbsp;
             <% } %>
 
+            <% if (locals.productName) { %>
+              <a href="<%- subscriptionPrivacyUrl %>" class="footer-link" data-l10n-id="subplat-privacy-notice">Privacy notice</a>
+              &nbsp;&nbsp;&bull;&nbsp;&nbsp;
```

Comment:
Man I really don't like this `&nbsp;&nbsp;&bull;&nbsp;&nbsp;` pattern or using `&bull;` as a separator for links but I know that's the pattern we set here in the old templates and it doesn't matter much 😆

Reply:
Same.